shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/segment/dml/predicate/value/PredicateCompareRightValue.java

```diff
@@ -28,6 +28,10 @@
 @Getter
 public final class PredicateCompareRightValue implements PredicateRightValue {
     
+    private final int startIndex;
```

Comment:
Do you think it is better to add `SQLSegment` interface for these `PredicateRightValue `?

Reply:
@tristaZero Now, PredicateRightValue implements SQLSegment.